### DISCUSSION TITLE
Assign Consumable To asset

### DISCUSSION THREAD

**Discussion by fiikra** [snipe/snipe-it#11932] [Ideas]
Hey i just remarque in my new installation, if we can add a checkout for consumable to asset for example if i have a shared printer and i supply it i should know that in the historic.
 so, any idea or suggestion I'm waiting thanks :D

**Comment by emreiyioz** [comment]
That would be a perfect feature! Thanks for the idea, I came to write the same idea here.

**Reply by emreiyioz** [reply]
Or would be also enough if we can checkout consumable to Locations...

**Comment by cletellier61** [comment]
Same need for us, it would be great thank you!
